tools/icu/configure.py

```diff
@@ -208,11 +208,11 @@ def extract_icu(icuroot, archive_path):
     shutil.rmtree(tempdir)
 
 def main():
-    chakra_icu_root = os.path.normpath(os.path.join(os.path.realpath(__file__), "..", "..", "deps", "Chakra.ICU"))
+    chakra_icu_root = os.path.normpath(os.path.join(os.path.realpath(__file__), "..", "..", "..", "deps", "Chakra.ICU"))
 
     argparser = ArgumentParser(description = "Download and set up ICU for use in ChakraCore")
     argparser.add_argument("-y", "--yes", action = "store_true", help = "Skip ICU License prompt text")
-    argparser.add_argument("version", help = "ICU version to download. Not compatible with --archive", default = "60.2", nargs = "?")
+    argparser.add_argument("version", help = "ICU version to download. Not compatible with --archive", default = "61.1", nargs = "?")
```

Comment:
Does this mean ChakraCore (in the CI builds) now uses v61.1 ?

Reply:
It means that if you run tools\icu\configure.py with no arguments, you will get 61.1. However, no Chakra-owned infrastructure invokes the script in this way. The only thing that uses this script on our end is our Linux and macOS binary release process which use ICU 57.1.

Reply:
For the time being, until we can figure out a good solution to avoid recompiling ICU on every CI run for each flavor, we havent turned on ICU in windows CI yet, and macOS and Linux CI use the system install of ICU. We were hoping to get Jenkins onto Server 2016 to get the Windows kit ICU, but that work was last touched in early march by @dilijev and I am not sure what the progress is. Our internal ChakraCore CI may be able to use some network share caching shenanigans to achieve something similar to the NuGet package we have with ChakraFull.

Reply:
There's now a PR out to update CI to RS4.